### ISSUE TITLE
Image, open in web browser option doesn't open web browser

### ISSUE THREAD

**Issue by bgilc** [hydrusnetwork/hydrus#1300]
### Hydrus version

509

### Qt major version

Qt 6

### Operating system

Windows 10

### Install method

Installer

### Bug description and reproduction

1. Start Hydrus.
2. Import some images.
3. From imported image list click right mouse on any image and use "open" -> "in web browser".
4. Windows image viewer displays the image instead of web browser.

**Comment by Zweibach** [comment]
I think you might need to set the browser path manually in `file -> options -> external programs`.  
Why? No idea.

**Comment by bgilc** [comment]
Yeah, that works. Can I provide any more info? It would be preferable to fix this for others. Or is this me only issue?
